issue.go

```diff
@@ -1512,3 +1512,27 @@ func (s *IssueService) AddRemoteLinkWithContext(ctx context.Context, issueID str
 func (s *IssueService) AddRemoteLink(issueID string, remotelink *RemoteLink) (*RemoteLink, *Response, error) {
 	return s.AddRemoteLinkWithContext(context.Background(), issueID, remotelink)
 }
+
+//Get all issues type with context
```

Comment:
Would you mind to add the API docs here as well? Similar to the other go docs?

Reply:
It will return with how many types of issue can be formed in Atlassian